dbuf/dbuf/aiofiles.py

```diff
@@ -0,0 +1,32 @@
+import concurrent
+import os
+import hailtop.utils as utils
+
+
+class AIOFiles:
```

Comment:
I'm kind of shocked this is needed.  Did you consider https://github.com/Tinche/aiofiles?

Reply:
This should probably go in hailtop.utils.

Reply:
There has not been a recent release of that project. My memory is vague at this point but it wasn't working, and it's trivial to roll my own.